custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -550,12 +550,12 @@ def run_test_predict(self):
             if self.target_type == TargetType.TRANSFORM:
                 target_df = df[self.options.target]
                 __target_temp = NamedTemporaryFile()
-                target_df.to_csv(__target_temp.name, index=False)
+                target_df.to_csv(__target_temp.name, index=False, line_terminator="\r\n")
```

Comment:
@scottp-ml Just to confirm, the idea here is that datasets prepared on Mac OS X or Linux systems will ignore the \r here, right?